docs/markdown/connecting-to-testnet.md

```diff
@@ -8,20 +8,23 @@ This tutorial describes how to:
 1. Configure and start `moneyd` to connect to an ILP router on the testnet
 2. Send value in XRP to a [payment pointer](https://interledger.org/rfcs/0026-payment-pointers/) on the testnet
 
+
 ## Before you begin
 For this tutorial you will need to:
- 1. Install `moneyd` and SPSP. You can learn how to install `moneyd` 
-and SPSP from the [Getting Started](getting-started.md) tutorial.
+
+1. Install `moneyd` and SPSP. You can learn how to install `moneyd` 
+and SPSP from the [Getting Started](getting-started.html) tutorial.
 2. Install the [moneyd XRP uplink](https://github.com/interledgerjs/moneyd#uplinks) using the command:
 ```shell
 npm install -g moneyd-uplink-xrp
 ```
 
-NOTE: Even though Interledger supports many different underlying asset types, you do not need to use real 
+**Note:** Even though Interledger supports many different underlying asset types, you do not need to use real 
 cryptocurrency for this tutorial. Since you will be sending value through an ILP router on the testnet, 
 funds will be sourced from the [XRP testnet faucet](https://xrpl.org/xrp-test-net-faucet.html) and 
 will have no real-world value.
 
+
 ## Configuring moneyd
 NOTE: Before configuring `moneyd`, make sure to delete the `.moneyd.test.json` config file, located in the home folder 
```

Comment:
New line here, too, please. Also, to be consistent the note should match the styling in line 22, i.e.:

> **Note:** Before configuring `moneyd`, make sure to delete the `.moneyd.test.json` config file, located in the home folder of your computer, if the file already exists. 

Also, another line break between 29 and 30.

Reply:
https://github.com/interledger/interledger.github.io/pull/115/files#diff-95b449acc2e692458b91372a5b6e33e7R32